### ISSUE TITLE
Any way to use pretrained model?

### ISSUE THREAD

**Issue by ishanshukla97** [Unity-Technologies/ml-agents#2039]
I find unity pretrained models (like WalkerLearning.nn) could save lots of training time for me. If I were to start from scratch, to achieve that mean reward I would have to train ~1 million steps. I would like to use this pretrained model to further train it for other tasks. 

Is it possible load the pretrained .nn file for training?

**Comment by roboserg** [comment]
it is possible, did you read the official documentation of this repo? You load the model with --load and set the max steps in config yaml to be greater than what the model already had.

**Comment by ervteng** [comment]
Hey @ishanshukla97, @roboserg, unfortunately it is actually not possible. You can load Tensorflow models, but not `.nn` models. There's currently no way to uncompile a .nn back to TF. 

Even the Walker models, though, don't take too long to train, maybe an hour tops.

**Comment by ishanshukla97** [comment]
Thanks for making it clear.

**Comment by ishanshukla97** [comment]
Well, tbh I'm training the walker model with predefined hyperparameters and looks like it will take more than a day.
